src/types/Firestore.ts

```diff
@@ -19,6 +19,10 @@ export interface Experience {
   tables: Record<string, Record<string, Table>>;
 }
 
+export interface Settings {
+  showChat?: boolean;
+}
```

Comment:
move this from types/Firestore into separate type file

Reply:
moved here: https://github.com/sparkletown/sparkle/pull/2261/commits/d42a8ec5d419219b9d48075684b1d1ceac94025b